docs/lists_degrees.md

```diff
@@ -122,10 +122,60 @@ Quality: Manually updated on an ad-hoc basis. Please submit a pull request if in
 |---|---|---|
 | `id` | UUID | A unique identifier. The same as `dttp_id` if that field is non-`nil`, otherwise a new UUID was minted at import time. |
 | `name` | string | The long name of the subject, eg "accountancy" |
-| `synonyms` | string array | A list of common alternative names |
+| `suggestion_synonyms` | string array | A list of common alternative names that *may* be appropriate for this subject |
+| `match_synonyms` | string array | A list of common alternative names that are equivalent to this subject. An answer matching a match synonym can be safely matched to this subject.|
 | `dttp_id` | uuid | The ID used for this subject in DTTP |
 | `hesa_itt_code` | string | The ID used for this subject in HESA |
 
+### `DfE::ReferenceData::Degrees::COMMON_COMPOUND_SUBJECTS`
+
+```ruby
+require 'dfe/reference_data/degrees'
+```
+
+Common combinations of degree subjects (Eg, subjects of the form "X with Y" or "X and Y")
+
+Owner: Apply team.
+
+Users: Apply team.
+
+Source: https://github.com/DFE-Digital/apply-for-teacher-training-prototype/blob/main/app/data/degree-subjects.js
+
+Quality: Manually updated on an ad-hoc basis. Please submit a pull request if inaccuracies or omissions are found.
+
+| Field | Type | Purpose |
+|---|---|---|
+| `id` | UUID | A unique identifier. The same as `dttp_id` if that field is non-`nil`, otherwise a new UUID was minted at import time. |
+| `name` | string | The long name of the subject, eg "accountancy" |
+| `suggestion_synonyms` | string array | A list of common alternative names that *may* be appropriate for this subject |
+| `match_synonyms` | string array | A list of common alternative names that are equivalent to this subject. An answer matching a match synonym can be safely matched to this subject.|
+| `components` | UUID array | The `SINGLE_SUBJECTS` IDs of the compound parts, in order |
```

Comment:
Suggest ending this in `_ids` to make it obvious that it contains IDs not objects, and that an array is expected. Might also be worth explaining what the "order" is.

```suggestion
| `subject_ids` | UUID array | The `SINGLE_SUBJECTS` IDs of the individual subjects, in the same order as they appear in the `name` |
```